src/promptflow-azure/promptflow/azure/_utils/general.py

```diff
@@ -47,3 +59,36 @@ def set_event_loop_policy():
         # Reference: https://stackoverflow.com/questions/45600579/asyncio-event-loop-is-closed-when-getting-loop
         # On Windows seems to be a problem with EventLoopPolicy, use this snippet to work around it
         asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+
+
+def is_workspace(workspace: Workspace) -> bool:
+    return workspace._kind == WorkspaceKind.DEFAULT
```

Comment:
These util functions are too trivial. Can put them in to the `WorkspaceKind` class like `def is_hub(kind)`  or `def is_hub(workspace)`

Reply:
sounds good to me, have updated one version